code/controllers/subsystem/mapping.dm

```diff
@@ -531,10 +531,11 @@ GLOBAL_LIST_EMPTY(the_station_areas)
 		message_admins("Failed to set new map with next_map.json for [change_to.map_name]! Using default as backup!")
 		return
 
-	if (change_to.config_min_users > 0 && GLOB.clients.len < change_to.config_min_users)
+	var/filter_threshold = get_active_player_count(alive_check = FALSE, afk_check = TRUE, human_check = FALSE)
+	if (change_to.config_min_users > 0 && filter_threshold != 0 && filter_threshold < change_to.config_min_users)
```

Comment:
Is the `filter_threshold != 0` check here necessary? What circumstances would have it be 0, and do we not want to log that we are entering delta with 0 active players?

Reply:
pre-round lobby it is 0.

Reply:
I think I have now forgotten why I asked that question so it's probably fine.